src/agent/src/rpc.rs

```diff
@@ -253,11 +253,14 @@ impl AgentService {
         if req.timeout == 0 {
             let s = Arc::clone(&self.sandbox);
             let mut sandbox = s.lock().await;
-            let ctr = sandbox
-                .get_container(&cid)
-                .ok_or_else(|| anyhow!("Invalid container id"))?;
 
-            ctr.destroy().await?;
+            sandbox.bind_watcher.remove_container(&cid).await;
```

Comment:
It's better to do the bind_watcher.remove_container after destroy the container, in case there's some cleanup in the bind watcher's volume depended on the container's destroy such as rootfs umount.